pages/linux/zypper.md

```diff
@@ -21,3 +22,7 @@
 - Search package via keyword:
 
 `zypper search {{keyword}}`
+
+- Show information related to configured repositories:
+
+`zypper repos -P`
```

Comment:
We prefer long argument format when is possible:
```suggestion
`zypper repos --sort-by-priority`
```
Why do you think `--sort-by-priority` is an essential flag for this command?

Reply:
When you run 
> zypper repos

zypper suggest to run zypper repos -P (or the long argument) to get more information. Just add a column with priority of each repo. Sometime it's useful to know the repo's priority

Reply:
Ok, apply the long version and we can merge this.